src/bindings/wasm/tvgWasmLottieAnimation.cpp

```diff
@@ -155,6 +158,56 @@ struct TvgWgEngine : TvgEngineMethod
     }
 };
 
+struct TvgGLEngine : TvgEngineMethod
+{
+    uintptr_t context = 0;
+    ~TvgGLEngine() override
+    {
+    #ifdef THORVG_GL_RASTER_SUPPORT
+        if (context) {
+            Initializer::term(tvg::CanvasEngine::Gl);
+            emscripten_webgl_destroy_context(context);
+            context = 0;
+        }
+    #endif
+    }
+
+    Canvas* init(string& selector) override
+    {
+    #ifdef THORVG_GL_RASTER_SUPPORT
+        EmscriptenWebGLContextAttributes attrs{};
+        attrs.alpha = true;
+        attrs.depth = false;
+        attrs.stencil = false;
+        attrs.premultipliedAlpha = true;
+        attrs.failIfMajorPerformanceCaveat = false;
+        attrs.majorVersion = 2;
+        attrs.minorVersion = 0;
+        attrs.enableExtensionsByDefault = true;
+
+
+        context = emscripten_webgl_create_context(selector.c_str(), &attrs);
+
+        if (context == 0) return {};
+
+        emscripten_webgl_make_context_current(context);
+
+        if(Initializer::init(0, tvg::CanvasEngine::Gl) != Result::Success) return {};
+
+        return GlCanvas::gen();
+    #endif
```

Comment:
How about put the line 195, 197 after `#endif` line.
It'd be better to ensure function return regardless of macro. Also for code consistency with the other engines.

Reference: line 149, 150